src/NumericalAlgorithms/Spectral/Spectral.hpp

```diff
@@ -131,6 +163,12 @@ Matrix interpolation_matrix(size_t num_points, const T& target_points) noexcept;
 template <Basis BasisType, Quadrature QuadratureType>
 const Matrix& spectral_to_grid_points_matrix(size_t num_points) noexcept;
 
+/*!
+ * \brief Vandermonde matrix for a one-dimensional mesh. See templated
```

Comment:
I'm not sure if "Vandermonde" is as helpful as "Matrix to go from nodal to modal (spectral) representation, i.e. the Vandermonde matrix."